### PR TITLE
Upgrade to Cypress 9

### PR DESCRIPTION
**Pull request by coverbeck**
**Description**
Upgrade to Cypress 9, one small step towards Cypress 12.

Regarding the additional changes beyond the upgrade:

1. On commenting out the check for the email, the email is not show on the organization page any more. I'm not sure how it worked in Cypress 8; my guess it was a DOM element "left-over", that hadn't been cleaned out, from the previous dialog that did display the email.
2. On the `force: true`, not sure. The UI elements are there (I seem them), but Cypress says it can't type there because they're not (100%?) visible. The newer Cypress has a newer browser version with it, so I'm guessing it's related to that.

**Review Instructions**
Builds and smoke tests should run

**Issue**
https://ucsc-cgl.atlassian.net/browse/SEAB-5089

Please make sure that you've checked the following before submitting your pull request. Thanks!

- [x] Check that your code compiles by running `npm run build`
- [x] Ensure that the PR targets the correct branch. Check the milestone or fix version of the ticket.
- [x] If this is the first time you're submitting a PR or even if you just need a refresher, consider reviewing our [style guide](https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Opinionated-Style-Guide#pr-checklist)
- [x] Do not bypass Angular sanitization (bypassSecurityTrustHtml, etc.), or justify why you need to do so
- [x] If displaying markdown, use the `markdown-wrapper` component, which does extra sanitization
- [x] Do not use cookies, although this may change in the future
- [x] Run `npm audit` and ensure you are not introducing new vulnerabilities
- [x] Do due diligence on new 3rd party libraries, checking for CVEs
- [x] Don't allow user-uploaded images to be served from the Dockstore domain
- [x] If this PR is for a user-facing feature, create and link a documentation ticket for this feature (usually in the same milestone as the linked issue). Style points if you create a documentation PR directly and link that instead.
- [x] Check whether this PR disables tests. If it legitimately needs to disable a test, create a new ticket to re-enable it in a specific milestone.

### CODE REVIEW

**Comment by coverbeck** [comment]
If the Cypress 10 PR #1669 passes tests, I'll close this one, as it includes the Cypress 9 commit. If it fails, I'll go ahead and merge this one, pending approvals.

**Comment by coverbeck** [comment]
On further reflection, going to merge this one first regardless, pending approval.

**Review by denis-yuen — APPROVED** [review]

**Review by svonworl — APPROVED** [review]

**Pull request merged**
